app/services/hyrax/workflow/changes_required_notification.rb

```diff
@@ -8,15 +12,24 @@ def subject
         end
 
         def message
-          "#{title} (#{work_id}) requires additional changes before approval.\n\n '#{comment}'"
+          "#{title} (#{link_to work_id, document_path}) requires additional changes before approval.\n\n '#{comment}'"
         end
 
       private
 
+        def solr_document
```

Comment:
Instead of loading the solr document, could we just add an accessor for the `entity` (from https://github.com/projecthydra-labs/hyrax/blob/master/app/services/hyrax/workflow/abstract_notification.rb#L11)?